A72-open-telemetry-tracing.md

```diff
@@ -105,94 +133,181 @@ class OpenTelemetryPluginBuilder {
+* If the RPC experienced name resolution delay, add an Event at the start of the
+  call with the name "Delayed name resolution complete" upon completion of the
+  name resolution process.
+* When the call is closed, set RPC status and end the parent span. gRPC status "OK" 
+  is recorded with status "OK", while other gRPC statuses are marked as "ERROR". 
+  Non-"OK" statuses include their code as a description, the same below.
 
 On attempt span:
-* When span is created, add attribute `previous-rpc-attempts` that captures the
-  number of preceding attempts for the RPC, and attribute `transparent-retry` that
-  shows whether stream is a transparent retry.
-* When the stream is created on transport, annotate delayed load balancer pick
-  complete, if any.
-* When an outbound message has been sent, add message events to capture seq no.,
-  type(SENT), uncompressed message size, and compressed message size if any
-  compression. The seq no. is a sequence of integer numbers starting from 0
-  to identify sent messages within the stream. The size is the total attempt message
-  bytes without encryption, not including grpc or transport framing bytes.
-* When an inbound message has been received from the transport, add message
-  events to capture seq no., type(Received), wire message size, and uncompressed
-  message size if any decompression. The seq no. is a sequence of integer numbers
-  starting from 0 to identify received messages within the stream.
+* When span is created, set the attribute with key `previous-rpc-attempts` and an 
+  integer value representing the count of previous attempts made for the RPC.
+* When span is created, set the attribute with key `transparent-retry` and a 
+  boolean value indicating whether the stream is undergoing a transparent retry.
+* If the RPC experienced load balancer pick delay, add an Event with the name 
+  "Delayed LB pick complete" upon creation of the stream on the transport. 
+* When the application sends an outbound message to the transport, add an Event 
+  with name "Outbound message sent" and the following attributes:
+  * key `message.event.type` with string value "SENT".
+  * key `message.message.id` with integer value of the seq no. The seq no. is a sequence
+    of integer numbers starting from 0 to identify sent messages within the stream, the same below.
+  * key `message.event.size.uncompressed` with integer value of uncompressed 
+    message size. The size is the total attempt message bytes without encryption,
+    not including grpc or transport framing bytes, the same below.
+  * If any compression, key `message.event.size.compressed` with integer value
+    of compressed message size.
+* When an inbound message has been received from the transport, add an Event
+  with name "Inbound message read" and the following attributes:
+  * key `message.event.type` with String value "RECEIVED".
+  * key `message.message.id` with integer value of the seq no.
```

Comment:
We should say this in a way that clarifies that the recv seq no is independent of the send seq no.  Suggest similar wording to above:

The seq no indicates the order of the received messages on the attempt (i.e., it starts at 0 and is incremented by 1 for each message received).

Same for server side.